website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -1,17 +1,31 @@
 ---
-title: Deploying a Subgraph to Subgraph Studio
+title: Deploy Using Subgraph Studio
 ---
 
-These are the steps to deploy your subgraph to Subgraph Studio:
+Learn how to deploy your subgraph in Subgraph Studio.
 
-- Install The Graph CLI (with either yarn or npm)
-- Create your Subgraph in Subgraph Studio
-- Authenticate your account from the CLI
-- Deploying a Subgraph to Subgraph Studio
+> Note: When you deploy a subgraph, you push it to Subgraph Studio, where you'll be able to test it. It's important to remember that deploying is not the same as publishing. When you publish a subgraph, you're publishing it on-chain.
 
-## Installing Graph CLI
+## Subgraph Studio Overview
 
-There is a CLI to deploy subgraphs to [Subgraph Studio](https://thegraph.com/studio/). Here are the commands to install `graph-cli`. This can be done using npm or yarn.
+In [Subgraph Studio](https://thegraph.com/studio/), you can do the following:
+
+- Create and manage your API keys for specific subgraphs
+- Create your subgraph through the Studio UI
+- Deploy your subgraph using the The Graph CLI
+- Integrate your subgraph in staging using the query URL
+- Publish your subgraph with the Studio UI
+- Test your subgraph in the playground environment
```

Comment:
I would put this one before the "integrate your subgraph in staging using the development query URL".